core/controllers/admin_test.py

```diff
@@ -69,14 +69,26 @@ class SampleMapReduceJobManager(jobs.BaseMapReduceOneOffJobManager):
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """Entity Classes to Map Over."""
```

Comment:
Only the first character of a sentence has to be caps (please fix this throughout the file)